### PR TITLE
[Cody GA Docs] Improvements to Cody Quickstart tutorial

### PR DESCRIPTION
**Pull request by MaedahBatool**
## Test plan

Docs added

## Preview 🤩
[Preview Link](https://docs.sourcegraph.com/@fix-qs)

### CODE REVIEW

File: doc/cody/quickstart.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 <p class="subtitle">In this quickstart guide, you'll learn how to use Cody once you have installed the extension in your VS Code editor. Here you will perform the following three tasks:</p>
 
-1. Try the `Generate Unit Tests` command
-2. Ask Cody to suggest bug fixes and changes to code snippets
-3. Ask Cody to pull additional information from the documentation
+1. Generate unit tests for your code
+2. Identify errors and suggest code fixes
```
Comment (taylorsperry) [inline doc/cody/quickstart.md:6]: "and get suggested code fixes"

File: doc/cody/quickstart.md [hunk 2]
```diff
@@ -14,11 +14,13 @@
 
 ## Getting started with Cody extension and commands
 
-After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's `Chat` panel will open. This interface is used to ask Cody questions and paste in code snippets.
+After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's **Chat** panel will open. This interface is used to ask Cody questions and paste in code snippets.
```
Comment (taylorsperry) [inline doc/cody/quickstart.md:17]: I think this refers to the old UX, no? The new UX won't open the chat panel automatically.

File: doc/cody/quickstart.md [hunk 3]
```diff
@@ -69,12 +71,12 @@ function getTimeAgoDescription(dateString) {
 
 ## 1. Generate a unit test
 
-To ensure code quality and early bug detection, one of the most useful commands that Cody offers is `Generate Unit Tests`. It quickly helps you write a test code for any snippet that you have highlighted. To generate a unit test for our example function:
+To ensure code quality and early bug detection, one of the most useful commands that Cody offers is **Generate unit tests**. It quickly helps you write a test code for any snippet that you have highlighted. To generate a unit test for our example function:
```
Comment (taylorsperry) [inline doc/cody/quickstart.md:74]: "helps you write ~a~ test code"

File: doc/cody/quickstart.md (hunk 2 above)
Comment (MaedahBatool) [reply doc/cody/quickstart.md:17]: Fixed it.

**Review by toolmantim — APPROVED** [review]

File: doc/cody/quickstart.md [hunk 4]
```diff
@@ -14,11 +14,13 @@
 
 ## Getting started with Cody extension and commands
 
-After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's `Chat` panel will open. This interface is used to ask Cody questions and paste in code snippets.
+After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's panel will open. This interface is used to start a **New Chat**, run Cody **commands** or get access to releavnt resources.
 
-Cody also supports `Commands` with VS Code. These are quick, ready-to-use prompt actions that you can apply to any code or text-based snippet you've highlighted. You can run a command in 3 ways:
+![Cody icon in side activity bar ](https://storage.googleapis.com/sourcegraph-assets/Docs/cody-quickstart/cody-icon-vscode.png)
```
Comment (kalanchan) [inline doc/cody/quickstart.md:19]: @MaedahBatool let's bump this to v0.18 at least, new search UI will be present

File: doc/cody/quickstart.md [hunk 5]
```diff
@@ -120,15 +122,13 @@ Leveraging the failed test output, Cody is able to identify the potential bug an
...
-
-Inside the Cody chat, type `/doc` followed by a query to search documentation. For example, ask Cody: "Can the getTimeAgoDescription() function run out of memory?"
+## 3. Ask Cody to add code documentation
 
-And you get the following response:
+Cody can also create well-crafted code documentation and comments, significantly improving the readability experience. You can add it by selecting a code snippet and clicking **Document** from the **Commands** panel. Alternatively, select the code, and inside the Cody chat window, type `/doc`. Cody will add a comment to the top of the code snippet.
 
-![Cody referencing docs ](https://storage.googleapis.com/sourcegraph-assets/Docs/cody-quickstart/get-ref-docs.png)
+<video width="1920" height="1080" loop playsinline controls style="width: 100%; height: auto; max-width: 50rem">
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/code-comments-cody.mp4" type="video/mp4">
```
Comment (kalanchan) [inline doc/cody/quickstart.md:130]: nice!

File: doc/cody/quickstart.md [hunk 6]
```diff
@@ -14,11 +14,13 @@
...
-After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's `Chat` panel will open. This interface is used to ask Cody questions and paste in code snippets.
+After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's panel will open. This interface is used to start a **New Chat**, run Cody **commands** or get access to releavnt resources.
 
-Cody also supports `Commands` with VS Code. These are quick, ready-to-use prompt actions that you can apply to any code or text-based snippet you've highlighted. You can run a command in 3 ways:
+![Cody icon in side activity bar ](https://storage.googleapis.com/sourcegraph-assets/Docs/cody-quickstart/cody-icon-vscode.png)
 
-1. Type `/` in the chat bar, and Cody will suggest a list of commands
+Cody supports **Commands** with VS Code. These are quick, ready-to-use prompt actions that you can apply to any code or text-based snippet you've highlighted. You can run a command in 3 ways:
+
+1. Click **New Chat** button, type `/` in the chat bar, and Cody will suggest a list of commands
```
Comment (kalanchan) [inline doc/cody/quickstart.md:23]: nit: maybe add an image here to showcase each of these trigger command operations?

**Review by kalanchan — CHANGES_REQUESTED** [review]
overall looks good, added some comments!

File: doc/cody/quickstart.md [hunk 7]
```diff
@@ -14,14 +14,18 @@
 
 ## Getting started with Cody extension and commands
 
-After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's `Chat` panel will open. This interface is used to ask Cody questions and paste in code snippets.
+After installing the extension, the side activity bar will display an icon for **Cody**. Click this icon, and Cody's panel will open. This interface is used to start a **New Chat**, run Cody **commands** or get access to releavnt resources.
 
-Cody also supports `Commands` with VS Code. These are quick, ready-to-use prompt actions that you can apply to any code or text-based snippet you've highlighted. You can run a command in 3 ways:
+![Cody icon in side activity bar ](https://storage.googleapis.com/sourcegraph-assets/Docs/cody-quickstart/cody-icon-vscode.png)
```
Comment (kalanchan) [inline doc/cody/quickstart.md:19]: it's still showing v0.14.5 :( 

could you update the image to show v0.18 so that the new search bar is present as well!

**Pull request merged**